inc/cleantalk-public-validate-skip-functions.php

```diff
@@ -0,0 +1,172 @@
+                 $_GET['wc-ajax'] === 'update_order_review' ||
+                 $_GET['wc-ajax'] === 'add_to_cart' ||
+                 $_GET['wc-ajax'] === 'remove_from_cart' ||
+                 $_GET['wc-ajax'] === 'get_variation' ||
+                 $_GET['wc-ajax'] === 'get_customer_location'
+             )) ||
+         /* END: WooCommerce Service Requests  */
+         apbct_is_in_uri('/wp-admin/') ||
+         apbct_is_in_uri('wp-login.php') ||
+         apbct_is_in_uri('wp-comments-post.php') ||
+         apbct_is_in_referer('/wp-admin/') ||
+         apbct_is_in_uri('/login/') ||
+         apbct_is_in_uri('?provider=facebook&') ||
+         (isset($_GET['ptype']) && $_GET['ptype'] === 'login') ||
+         isset($_POST['ct_checkjs_register_form']) ||
+         (isset($_POST['signup_username'], $_POST['signup_password_confirm']) && isset($_POST['signup_submit'])) ||
+         $apbct->settings['forms__general_contact_forms_test'] == 0 ||
+         isset($_POST['bbp_topic_content']) ||
+         isset($_POST['bbp_reply_content']) ||
+         isset($_POST['fscf_submitted']) ||
+         (isset($_POST['log'], $_POST['pwd'], $_POST['wp-submit'])) ||
+         apbct_is_in_uri('/wc-api') ||
+         apbct_is_in_uri('wc-api=WC_Gateway_Tpay_Basic') || // Tpay payment Gateway plugin
+         (isset($_POST['wc_reset_password'], $_POST['_wpnonce'], $_POST['_wp_http_referer'])) || //WooCommerce recovery password form
+         (isset($_POST['woocommerce-login-nonce'], $_POST['login'], $_POST['password'], $_POST['_wp_http_referer'])) || //WooCommerce login form
+         (isset($_POST['provider'], $_POST['authcode']) && $_POST['provider'] === 'Two_Factor_Totp') || //TwoFactor authorization
+         (isset($_GET['wc-ajax']) && $_GET['wc-ajax'] === 'sa_wc_buy_now_get_ajax_buy_now_button') || //BuyNow add to cart
+         apbct_is_in_uri('/wp-json/wpstatistics/v1/hit') || //WPStatistics
+         (isset($_POST['ihcaction']) && $_POST['ihcaction'] === 'login') || //Skip login form
+         (isset($_POST['action']) && $_POST['action'] === 'infinite_scroll') || //Scroll
+         isset($_POST['gform_submit']) || //Skip gravity checking because of direct integration
+         (isset($_POST['lrm_action']) && $_POST['lrm_action'] === 'login') || //Skip login form
+         apbct_is_in_uri('xmlrpc.php?for=jetpack') ||
+         apbct_is_in_uri('connector=bridge&task=put_sql') ||
+         Server::inUri('cleantalk-antispam/v1/alt_sessions') || // Skip test for alt sessions
+         (apbct_is_in_uri('bvMethod=') && apbct_is_in_uri('bvVersion=') && isset($_POST['apipage']) && $_POST['apipage'] === 'blogvault') ||
+         (isset($_POST['wpstg-username'], $_POST['wpstg-pass'], $_POST['wpstg-submit']) && $_POST['wpstg-submit'] === 'Log In') //Accept Stripe Payments
+    ) {
+        return true;
+    }
+
+    return false;
+}
+
+/**
+ * Function for skip in ct_contact_form_validate()
+ * @return bool
+ */
+function skip_for_ct_contact_form_validate()
```

Comment:
Can we use more friendly names? Function name should reflect what it does. FE: if ( ct_contact_form_validate_skipped() ){}